charts/keda/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: keda
+appVersion: 0.0.0-snapshot
```

Comment:
Should we use what keda uses for version (currently)? maybe that's that, idk.
Same for the icon

Reply:
the official keda chart uses the gitsha as the app version. Whatever we use, I'd like to not have to maintain the value by hand.

```
$ helm search keda
NAME                    CHART VERSION                           APP VERSION     DESCRIPTION                                                 
kedacore/keda-edge      0.0.1-2019.09.26.18.17.46-dc6f360       dc6f360         Event based autoscaler for Azure Functions deployments on...
```